store/storetest/thread_store.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetest
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/store"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestThreadStore(t *testing.T, ss store.Store, s SqlSupplier) {
+	t.Run("SaveMultiple", func(t *testing.T) { testThreadStoreSaveMultiple(t, ss) })
```

Comment:
Test name and test function name should be renamed after addition of additional tests in 2b365a8884dd5f5e87783338c852fcd89d01f655